mmv1/third_party/terraform/services/bigquery/resource_bigquery_table_test.go.tmpl

```diff
@@ -2200,6 +2200,14 @@ resource "google_bigquery_table" "test" {
   {
     "name": "some_int",
     "type": "INTEGER"
+  },
+  {
+	"name": "reserved_word_for",
+	"type": "STRING"
```

Comment:
I added a test case with reserved words just in case, although it did not originally cause an error.

Reply:
Can you align the indentation of this new block and the one below with the rest? i.e. using space instead of tab

Reply:
```suggestion
  "name": "reserved_word_for",
  "type": "STRING"
```